plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/DatabaseNavigatorTree.java

```diff
@@ -416,6 +417,14 @@ public CheckboxTreeViewer getCheckboxViewer() {
         return (CheckboxTreeViewer) treeViewer;
     }
 
+    public boolean isFilterActive() {
+        return treeFilter != null && treeFilter.isActive();
+    }
+
+    public boolean isMatchingNeeded(Object element) {
+        return treeFilter.isMatchingNeeded(element);
+    }
```

Comment:
Why creating a new method for just a one call? I would just add this call inplace